frontend/src/voice-ai-lib/multimodal-live-client.ts

```
@@ -0,0 +1,320 @@
+      if (isTurnComplete(serverContent)) {
+        this.log('server.send', 'turnComplete');
+        this.emit('turncomplete');
+        //plausible theres more to the message, continue
+      }
+
+      if (isModelTurn(serverContent)) {
+        let parts: Part[] = serverContent.modelTurn.parts;
+
+        // when its audio that is returned for modelTurn
+        const audioParts = parts.filter(
+          (p) => p.inlineData && p.inlineData.mimeType.startsWith('audio/pcm')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)